ifrs17-template/Initialization/InitSystemorphBaseToMemory.ipynb

```diff
@@ -85,37 +85,7 @@
         {
             "cell_type": "code",
             "source": [
-                "await Import.FromFile(pathToImport + \"DataNodes/DataNodes_FR.csv\")",
-                "\n            .WithFormat(ImportFormats.DataNode)",
-                "\n            .WithTarget(DataSource)",
-                "\n            .WithActivityLog()",
-                "\n            .ExecuteAsync()"
-            ],
-            "metadata": {},
-            "execution_count": 0,
-            "outputs": []
-        },
-        {
-            "cell_type": "code",
-            "source": [
-                "await Import.FromFile(pathToImport + \"DataNodes/DataNodeStates_FR_2020_12.csv\")",
-                "\n            .WithFormat(ImportFormats.DataNodeState)",
-                "\n            .WithTarget(DataSource)",
-                "\n            .WithActivityLog()",
-                "\n            .ExecuteAsync()"
-            ],
-            "metadata": {},
-            "execution_count": 0,
-            "outputs": []
-        },
-        {
-            "cell_type": "code",
-            "source": [
-                "await Import.FromFile(pathToImport + \"DataNodes/DataNodeParameters_FR_2020_12.csv\")",
-                "\n            .WithFormat(ImportFormats.DataNodeParameter)",
-                "\n            .WithTarget(DataSource)",
-                "\n            .WithActivityLog()",
-                "\n            .ExecuteAsync()"
+                "#!eval-notebook \"InitSystemorphBaseToMemoryFr\""
```

Comment:
Is this needed?

Reply:
As discussed in the call, I will re-arrange the Init into 3 different files by the data nodes (Fr, Ch, Es). The separation between the base and the import will be kept only for the Ch.

Reply:
Now it is completely obsolete, so I will mute this thread.